actionpack/CHANGELOG.md

```diff
@@ -2,4 +2,8 @@
 
     *Hartley McGuire*
 
+*   Add `singular` Boolean variable to `polymorphic_routes` so that we can generate proper URLS for nested and non nested singular resources
```

Comment:
The actual implementation probably isn't that relevant here.
We can describe the behaviour that changed.
Maybe something like the following, although I'm not sure that is the actual change :)

````suggestion
*   Use the correct path when the resource is singular

    Before:
    ```
    edit_author_path
    # => /author.1
    ```
    
    After:
   ```ruby
    edit_author_path
    # => /author
   ```
````

Reply:
Adding this change! Thank you